images/access_setup/README.md

```diff
@@ -0,0 +1,44 @@
+# Access Setup
+
+## Goals
+Given a new KCP instance or compute cluster, deploy the minimum
+amount of resources required to operate that cluster from a git repository
+implementing a CD process (c.f. gitops/sre). The CD process will then be
+responsible for installing/registering all the required resources.
+
+This action needs to be performed only once in the lifetime of the resource
+being initialized.
+
+## Definitions
+
+* Pipelines Service SRE team: The team responsible for deploying and managing
+  Pipelines Service on one or more KCP instances and workspaces.
+
+## KCP
+`setup_kcp.sh` needs to be run once by the Pipelines Service SRE team for each
+`KCP instance`/`KCP organization` pair operated by the Pipelines Service SRE team.
+
+The script will:
+1. Create the `$KCP_WORKSPACE` workspace in the organization workspace.
+2. Create a `pac-manager` serviceaccount in the new workspace, in the `pipelines-as-code` namespace.
+3. Generate the kubeconfig for the serviceaccount.
+
+The generated kubeconfig must then be placed in the `credentials/kubeconfig/kcp`
+folder of the SRE gitops repository to enable the automation of the compute
+registration to kcp.
+
```

Comment:
It would help to add a example like this right here:

`./setup_kcp.sh --kubeconfig /kcp/kcpinstance.kubeconfig --kcp_org plnsvc-ws --work_dir /repo/sre `

Reply:
Fixed